gateway/1.5/modules/ROOT/pages/policies-pdk-debug-local.adoc

```diff
@@ -146,3 +145,17 @@ curl --location --request POST 'http://0.0.0.0:8081/some/route' \
 ----
 
 The backend service returns an echo response including the requests’s route, body, and headers sent with any modification performed by the custom policy or additional policies. Any logs included in the custom policy's source code are visible in the terminal running the containers.
+
+== Loglevel
+
+By default, the PDK debugging environment enables logs with the `debug` loglevel configured.
+
+To change the loglevel, edit the `logging.runtimeLogs.logLevel` value in `test/config/logging.yaml`.
+
+To learn more about editing a Flex Gateway's logging configuration file, see xref:flex-local-third-party-logs-config.adoc[].
+
+== Use the Rust Debugger
+
+Visual Studio Code (VS Code) includes a Rust debugger that can run unit tests. You cannot use the VS Code Debugger while the policy is deployed on a Flex Gateway. Meaning that to create the API mocking, you must use unit tests for the specific logic you need to debug.
+
+To debug Rust code in VS Code, you must install the https://marketplace.visualstudio.com/items?itemName=vadimcn.vscode-lldb[CodeLLDB Debugger Extension ^] or a different VS Code extension with the same functionality. To learn more about debugging Rust in VS Code, see https://code.visualstudio.com/docs/languages/rust#_debugging[Rust in Visual Studio Code ^].
```

Comment:
Rich text preview shows a space between the final link and the period at the end of that sentence. I'd test (if you haven't already) to make sure the xref preview text and carets actually need spaces between them.